### DISCUSSION TITLE
How to change the color of the search count text?

### DISCUSSION THREAD

**Discussion by danhngo-lx** [folke/noice.nvim#914] [Q&A] [answered]
Hi everyone, how can I change the color of the search count text? Here's what's I mean:
![image](https://github.com/user-attachments/assets/6663d53b-7beb-4540-9644-a8d8b3c5276d)

**Comment by ilias777** [comment, marked as answer]
It's `NoiceVirtualText`

**Reply by danhngo-lx** [reply]
It works. Thank you!
